File: easy/3190. Find Minimum Operations to Make All Elements Divisible by Three/main.java
// 3190. Find Minimum Operations to Make All Elements Divisible by Three
class Solution {
    public int minimumOperations(int[] nums) {
        int operations = 0;
        for (int num : nums) {
            int remainder = num % 3;
            operations += Math.min(remainder, 3 - remainder);
        }
        return operations;
    }
}

File: easy/3190. Find Minimum Operations to Make All Elements Divisible by Three/main.rs
// 3190. Find Minimum Operations to Make All Elements Divisible by Three
impl Solution {
    pub fn minimum_operations(nums: Vec<i32>) -> i32 {
        let mut operations = 0;
        for &num in &nums {
            let remainder = num % 3;
            operations += remainder.min(3 - remainder);
        }
        operations
    }
}